db/schema.rb

```diff
@@ -11,7 +11,16 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150317014350) do
+ActiveRecord::Schema.define(version: 20150413015135) do
+
+  create_table "forms", force: :cascade do |t|
+    t.text     "school"
+    t.integer  "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```

Comment:
I think you also want a foreign key to the users table.  Each student is also a user correct?